src/org/team4909/bionicframework/hardware/PotentiometerController.java

```diff
@@ -0,0 +1,58 @@
+package org.team4909.bionicframework.hardware;
+
+import edu.wpi.first.wpilibj.AnalogPotentiometer;
+import edu.wpi.first.wpilibj.SpeedController;
+import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj.command.InstantCommand;
+import edu.wpi.first.wpilibj.command.PIDSubsystem;
+
+public class PotentiometerController extends PIDSubsystem {
+	private SpeedController motor;
+	private AnalogPotentiometer potentiometer;
+	private double max;
+
+	public PotentiometerController(SpeedController motor, AnalogPotentiometer potentiometer, double p, double i, double d) {
+		super(p, i, d);
+
+		this.motor = motor;
+		this.potentiometer = potentiometer;
+
+		getPIDController().setContinuous(false);
+		
+		getPIDController().enable();
```

Comment:
When do you ever .disable() the pid controller?

Reply:
we don't. it's constantly held at angle.

Reply:
think we should have a way to easily disable?